docs/how-to-run.adoc

```diff
@@ -433,7 +436,27 @@ Where did 192.168.1.20 come from?  That's the external address of the external
 DNS server.  We knew that because it's listed in the `external_dns_ips` entry of
 the `config-rss.toml` file we're using.
 
-Having looked this up, the easiest thing will be to use `http://192.168.1.21` for your URL (replacing with `https` if you used a certificate, and replacing that IP if needed).  If you've set up networking right, you should be able to reach this from your web browser.  You may have to instruct the browser to accept a self-signed TLS certificate.  See also <<_connecting_securely_with_tls_using_the_cli>>.
+Having looked this up, the easiest thing will be to use `http://192.168.1.21` for your URL (replacing with `https` if you used a certificate, and replacing that IP if needed).  If you've set up networking right, you should be able to reach this from your web browser.  You may have to instruct the browser to accept a self-signed TLS certificate.  See also <<connecting-securely-with-tls-using-the-cli>>.
+
+=== Setting up an SSH tunnel for console access
+
+If you set up a fake external network (method 2 in <<external-networking>>), one
+way to be able to access the console of your deployment is by setting up an SSH
+tunnel. Console access is required to use the CLI for device authentication.
+The following is an example of how to access the console with an SSH tunnel.
+
+Nexus serves the console, so first get a nexus IP from the instructions above.
+
+On the machine from which you which to access the console, set up an SSH tunnel.
+This is an example for the lab machine `dunkin`. Replace the hostname there as
+appropriate, as well as the nexus IP.
```

Comment:
```suggestion
In our example, Omicron is running on the lab machine `dunkin`.  You'll usually want to set up the tunnel from the machine where you run your browser to the machine running Omicron.  In our case, we'd run this on the machine running the browser:
```

(I found some of the existing paragraph a little confusing)

Reply:
added something similar in f3ca0b4